woocommerce/woocommerce-ios
# Add the ability to create real screenshots

**To Test**
1. Have a WordPress.com account that has products, orders, and reviews (and doesn't use 2FA)
2. Add the login details to `WooCommerceScreenshots.swift`
3. Run `bundle exec fastlane snapshot` in the root directory of the project.
4. Wait a long time
5. In the `fastlane/screenshots` directory, you should now have a large number of localized screenshots suitable for embedding in device frames.
6. To test putting everything in device frames for use in the App Store, run `bundle exec fastlane create_promo_screenshots – the screenshots should be in proper device frames and match what's on the App Store.

Update release notes:
